lib/collectionspace/converter/publicart/config.yml

```diff
@@ -129,31 +129,45 @@ registered_profiles:
       Authorities:
         Organization:
         - currentowner
-  person:
+  organization:
     type: Authorities
     enabled: true
     identify_by_column: termdisplayname
     required_headers:
       - termdisplayname
     config:
-      mapper: PublicArtPerson
+      mapper: PublicArtOrganization
       name_field: termdisplayname
-      authority_type: Person
-      authority_subtype: person
-      Authorities: []
-      Vocabularies: []
-  organization:
+      authority_type: Organization
+      authority_subtype: organization
+      Authorities:
+        - name_field: currentplace
+          authority_type: Place
+          authority_subtype: place
+        - name_field: foundingplacelocal
+          authority_type: Place
+          authority_subtype: place
+        - name_field: foundingplaceshared
+          authority_type: Place
+          authority_subtype: place  
```

Comment:
@Paulmulonzia - the authority_subtype needs to be the `shortIdentifier` for the given authority. I usually look this up using the API: https://publicart.dev.collectionspace.org/cspace-services/placeauthorities  

It is also found in the refName URN of a term in the authority, in parentheses after `{*}authorities:name`

so this should be `place_shared`